fastlane-env-reminder/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "fastlane-env-reminder-action",
+  "version": "0.0.0",
```

Comment:
correct?

Reply:
Yes, that is correct 👍 It points to a workspace version.